news/news-4.0.0-beta-1.md

```diff
@@ -65,15 +65,9 @@ Maintainer highlights include:
 * Added eslint / prettier / stylelint tooling.
 * Uses torrent-get's 'table' mode for more efficient RPC calls.
 
-### BitTorrent 2.0 File Support
-
-- Transmission 4.0.0 now supports .torrent files that were created with the [BitTorrent v2 spec](http://bittorrent.org/beps/bep_0052.html). 4.0.0 users will be able to download and seed torrents created by both BitTorrent v1 and v2. Support for _creating_ v2 torrents is slated for the next major release of Transmission.
-
-### New Features
-
-- Support setting "default" trackers that can be used for announcing all public torrents
-- Newly-added seeds can start immediately and verify pieces as needed, instead of requiring a full verify before seeding can begin ([#2626](https://github.com/transmission/transmission/pull/2626))
+## Website Redesign
 
+The Transmission website at https://transmissionbt.com/ has been redesigned. Thanks to [@shatteredsite](https://github.com/shatteredsite) for doing the work, to [@Oleg-Chashko](https://github.com/Oleg-Chashko) for providing feedback, and to [@GaryElshaw](https://github.com/GaryElshaw) for feedback and for the initial suggestion!
```

Comment:
Wow! I haven't opened it for a while. Looks much more modern and responsive :tada: